### PR TITLE
Resolve mismatch between label width and alignment

### PR DESCRIPTION
**Pull request by DJMcNab**
If the alignment was set previously, and the `CrossAxisAlignment` was not `Start`, the width Parley believed we were in wouldn't match the width used for Masonry layout. This would lead to weird behaviour.

See the diff for the test images in https://github.com/linebender/xilem/commit/0a68159869243b2d9b3a5e58a7454b572eba7b2d for more context

### CODE REVIEW

File: xilem/src/view/prose.rs [hunk 1]
```diff
@@ -13,15 +13,26 @@ pub fn prose(content: impl Into<ArcStr>) -> Prose {
...
+        line_break_mode: LineBreaking::WordWrap,
     }
 }
 
+/// A version of [`prose`] suitable for including in the same line
+/// as other content.
+///
+/// Note that setting [`alignment`](Prose::alignment) on the result
+/// will be meaningless.
+pub fn inline_prose(content: impl Into<ArcStr>) -> Prose {
```
Comment (Philipp-M) [inline xilem/src/view/prose.rs:25]: Maybe `span` instead as I think it's shorter and similar to the web equivalent?

But `inline_prose` on the other hand is descriptive.

**Review by Philipp-M — APPROVED** [review]
Apart from nits, looks good to me.

**Review by jaredoconnell — APPROVED** [review]
Looks good. I appreciate the improved documentation. The tests look useful. I tested the http_cats example, and it works well.

File: masonry/src/widget/prose.rs [hunk 2]
```diff
@@ -11,7 +11,7 @@ use vello::peniko::BlendMode;
 use vello::Scene;
 
 use crate::text::{ArcStr, TextBrush, TextWithSelection};
-use crate::widget::label::LABEL_X_PADDING;
+use crate::widget::label::PROSE_X_PADDING;
```
[the remark this answers is not in the document]
Comment (DJMcNab) [reply masonry/src/widget/prose.rs:14]: Lol, I didn't realise I re-used the import from both. I'll address it.

**Pull request merged**